src/client/api/restful/get_outputs_id.h

```diff
@@ -11,6 +11,46 @@
 #include "core/address.h"
 #include "core/types.h"
 
+#define INDEXER_OUTPUTS_API_PATH "/api/plugins/indexer/v1/outputs"
+#define INDEXER_ALIASES_API_PATH "/api/plugins/indexer/v1/aliases"
+#define INDEXER_NFT_API_PATH "/api/plugins/indexer/v1/nfts"
+#define INDEXER_FOUNDRY_API_PATH "/api/plugins/indexer/v1/foundries"
+
+/**
+ * @brief All Query Params Type
+ *
+ */
+typedef enum {
+  QUERY_PARAM_ADDRESS = 0,    ///< The Bech32-encoded address that should be used to query outputs
+  QUERY_PARAM_DUST_RET,       ///< The presence of dust return unlock condition
```

Comment:
more specifically, it's a boolean.
```suggestion
  QUERY_PARAM_HAS_DUST_RET,       ///< The presence of dust return unlock condition
```

Reply:
Done.